test/install.mk

```diff
@@ -5,6 +5,10 @@
 
 INSTALL_OPTS="--set global.istioNamespace=${ISTIO_NS} --set global.configNamespace=${ISTIO_NS} --set global.telemetryNamespace=${ISTIO_NS} --set global.policyNamespace=${ISTIO_NS}"
 
+ifneq (${ENABLE_NAMESPACES_BY_DEFAULT}, false)
```

Comment:
I think that is 
ENABLE_NAMESPACES_BY_DEFAULT ?= true

Reply:
done